src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/PackageCodeAggregator.cs

```diff
@@ -77,6 +78,10 @@ internal record PackageCodeChange(long StateVersion)
     public CoreModel.PackageCodeOriginalCodeEntrySubstate? PackageCodeOriginalCode { get; set; }
 
     public CoreModel.PackageCodeVmTypeEntrySubstate? PackageCodeVmType { get; set; }
+
+    public bool CodeVmTypeIsDeleted { get; set; } = false;
+
+    public bool PackageCodeIsDeleted { get; set; } = false;
```

Comment:
`= false` - is there any reason we explicitly defined those?

Reply:
nope just readability, i'll remove then.